ui/src/pages/PodcastDetailPage.tsx

```diff
@@ -75,59 +78,63 @@ export const PodcastDetailPage = () => {
+                                .then(() => {
+                                    console.log("Refreshed")
                                 })
-                                }
-                        </div>
-                        }
+                        }}>refresh</span>
                     </div>
+
+                    <span className="block text-stone-500">{currentPodcast.author}</span>
+
+                    {<div className="flex gap-2">
+                        {currentPodcast.keywords && currentPodcast.keywords?.split(',').map((keyword, index) => {
+                            return <Chip key={index} index={index}>{keyword}</Chip>
+                        })}
+                    </div>}
+
+                    <span className="material-symbols-outlined inline cursor-pointer text-stone-800 hover:text-stone-600" onClick={()=>{window.open(configModel?.rssFeed+"/"+params.id)}}>rss_feed</span>
+                </div>
+
+                <div className="self-start flex gap-3 items-center">
+                    <span className="text-xs text-stone-500">{t('active')}</span>
+
+                    <Switcher checked={currentPodcast.active} setChecked={()=>{
+                        axios.put(apiURL + "/podcast/" + params.id + "/active")
+                            .then(() => {
+                                dispatch(setCurrentPodcast({...currentPodcast, active: !currentPodcast?.active}))
+                            })
+                    }}/>
                 </div>
             </div>
-            {currentPodcast.summary&&<div className="relative m-2">
-                <div id="summary" className={`podcast-summary ${lineClamp?'line-clamp-3':''}`} dangerouslySetInnerHTML={removeHTML(currentPodcast.summary)}/>
-                {(isOverflown('summary')||lineClamp)&&<div className="left-0 text-slate-600 underline  cursor-pointer "  onClick={()=>{
+
+            {currentPodcast.summary&&<div className="relative leading-[1.75] mb-8 text-sm text-stone-900">
+                <div id="summary" className={lineClamp?'line-clamp-3':''} dangerouslySetInnerHTML={removeHTML(currentPodcast.summary)}/>
+                {(isOverflown('summary')||lineClamp)&&<div className="cursor-pointer underline text-mustard-600 hover:text-mustard-500"  onClick={()=>{
                     setLineClamp(!lineClamp)
-                }}>{lineClamp?t('show-more'):t('show-less')}</div>}
-            </div>
-            }
-            <hr className="border-gray-400"/>
+                }}>
+                    {lineClamp?t('show-more'):t('show-less')}
+                </div>}
+            </div>}
+
             <div>
-                {
-                    selectedEpisodes.map((episode, index) => {
-                        return <PodcastDetailItem episode={episode} key={index} index={index} episodesLength={selectedEpisodes.length}/>
-                    })
-                }
+                <Heading2 className="mb-8">Available Episodes</Heading2>
```

Comment:
There is a missing translation.

Reply:
Thanks, fixed. If you can, please double-check the translations, as I just used Google Translate.